mlflow/R/mlflow/Dockerfile.dev

```diff
@@ -1,4 +1,4 @@
-FROM rocker/r-ver:4.1.2
+FROM r-base:latest
```

Comment:
Uses a debian base image that is compatible with the Docker installation on our CI systems